### DISCUSSION TITLE
non-return damper

### DISCUSSION THREAD

**Discussion by ESH-y** [firemodels/fds#13454] [Q&A]
Hi,
I need to model a non-return damper in order to prevent a situation where the smoke will enter the air shaft and pass between the floors. 
I need to model a damper that will open in case and the pressure difference between the pressure inside the shaft and the pressure outside the shaft will be 100 pascal. And in addition, the damper will remain open as long as the flow speed is above 3.

Does anyone have an idea how to model something like this?
How can the opening and closing of the damper be controlled according to pressures and/or flow speed?

I would appreciate any help
Thank you very much

**Comment by drjfloyd** [comment]
User Guide Chapter 18 Devices and Control Logic

**Reply by ESH-y** [reply]
hi
thank a lot for your answer
i have been reading this chapter allready and still i have no idea how to write that in fds.
please can you help me?

**Reply by drjfloyd** [reply]
You need to come up with some set of contorl functions and devices that give you the behavior you want. For example you want to use pressure and velocity so you will need devices to measure those.  I suggest you work backwards and develop a logic diagram of your system.

-At the end you want a single control function that opens/closes a damper.  
-You want that function to be true if either the pressure is greater than 100 or the flow is over 3 m/s. "or" should suggest to you the ANY control function type.

Keep working backwards until you reach the basic measurements being made by devices.

**Reply by drjfloyd** [reply]
I also suggest you start with a simple fds input file that runs quickly and not a large complex geometry. It will probably take you multiple tries to get the logic correct and testing with an input that runs in a minute or two will be much easier than an input that takes hours to reach the point where your logic starts to be used.
